### ISSUE TITLE
OSPF redistribute

### ISSUE THREAD

**Issue by JBaptisme** [cisco/cisco-network-puppet-module#531]
Hello,

I'm trying to configure redistribution of direct route into OSPF.
I can't find any directive for do that, Is it normal ?

For example I want to generate configuration lile:

```
router ospf TOTO
router-id 192.168.1.10
redistribute direct route-map myroutemap
```

Is  mandatory to use cisco_command_config ?

Thanks for help

**Comment by mikewiebe** [comment]
@JBaptisme You are right.  That is missing from our current support.  It's not currently on our roadmap of things to add but you can use command_config as a stop gap.  We love contributions to the module.  Is this something you would be interested in or willing to add?

**Comment by mikewiebe** [comment]
@JBaptisme The PR above adds redistribution support for OSPF.
